### ISSUE TITLE
[Bug]: 21:9 Screen Format UI Issue

### ISSUE THREAD

**Issue by formeo14** [streamyfin/streamyfin#299]
### What happened?

When using Streamyfin on an Android device with a 21:9 aspect ratio screen, some buttons are positioned too deep within the UI, making them difficult to access or interact with. This issue affects the overall usability of the app on devices with this screen format.

### Reproduction steps

1. Open Streamyfin on an Android device with a 21:9 aspect ratio screen (e.g., Samsung Galaxy S21 Ultra).

### Which device and operating system are you using?

Android 14
Device with 21:9 screen format

### Version

0.22.0

### If applicable, please add screenshots to help explain your problem. You can drag and drop images here or paste them directly into the comment box.

![Screenshot (1)](https://github.com/user-attachments/assets/c6556a40-1c3e-4a1f-b6f2-8316e526a265)
![Screenshot (2)](https://github.com/user-attachments/assets/0089d3bd-a2e5-48a2-8c9d-1f693fabe77d)
![Screenshot (3)](https://github.com/user-attachments/assets/436a7c7c-5b5f-451c-9b3b-bb1bbd4b6940)
![Screenshot (4)](https://github.com/user-attachments/assets/9fa93b48-a507-4093-bdcd-c1861817726b)
![Screenshot (5)](https://github.com/user-attachments/assets/1ad0dd5f-845a-4244-835a-78390f5d986c)

**Comment by fredrikburmester** [comment]
Is does not look like you are on the latest version of the app. Please update first. Some of the issues might be fixed. 

There's also a setting now that enables/disabled safe areas in the video controls.

**Comment by formeo14** [comment]
> Is does not look like you are on the latest version of the app. Please update first. Some of the issues might be fixed.
> 
> There's also a setting now that enables/disabled safe areas in the video controls.

No, the issues are still present. The player has been changed, but I created a separate issue for that since it doesn’t align with my camera cutout. The rest of the player seems fine, except for the gray overlay, which should stop at the top and bottom, and the buttons aren’t functioning properly—only the close button works.

![Screenshot_20250101-190312_Streamyfin](https://github.com/user-attachments/assets/81536728-8146-4e00-adc0-2ceba1e7368f)
![Screenshot_20250101-190825_Streamyfin](https://github.com/user-attachments/assets/9a4f9dd6-47ef-4afa-af07-9564f9236da9)

**Comment by formeo14** [comment]
![Screenshot_20250104-132353_Streamyfin](https://github.com/user-attachments/assets/d9783df0-b220-4da5-9e0b-25eecbe3ce8f)
Now it respects the camera a bit too good v. 23.0.0
Horizontal does work,  vertical does now not work at all😅

**Comment by topiga** [comment]
Hey ! Can you update us on your experience ? Is it still an issue ?

**Comment by formeo14** [comment]
No works fine now thank you!
